sources/packages/web/src/views/aest/student/SINManagement.vue

```diff
@@ -164,18 +125,17 @@ export default {
     watch(props.studentId, loadSINValidations, { immediate: true });
 
     const addNewSIN = async () => {
-      const modalResult = await addNewSINModal.value.showModal();
-      if (!modalResult) {
-        return;
-      }
+      const addNewSINData = await addNewSINModal.value.showModal();
+      if (addNewSINData)
+        createNewSIN(addNewSINData as CreateSINValidationAPIInDTO);
```

Comment:
Can we add the await here for consistency?

Reply:
removed the unwanted method